web/src/components/navbar/NavBar.test.tsx

```diff
@@ -94,6 +94,30 @@ describe("<NavBar />", () => {
     useMockRouter({});
   });
 
+  it("shows business name when showSidebar is true", () => {
+    useMockUserData(generateBusinessNamedUserData());
+    setLargeScreen(false);
+    render(
+      withAuth(<NavBar landingPage={false} task={undefined} showSidebar={true} />, {
+        isAuthenticated: IsAuthenticated.TRUE,
+      })
+    );
+
+    expect(screen.getByText(businessName)).toBeInTheDocument();
+  });
+
+  it("does not show business name when showSidebar is false", () => {
```

Comment:
[Pebble] Since are setting the media query to false and rendering a mobile view, we should update the test name so it clearly states this is in mobile